Makefile

```diff
@@ -77,8 +77,10 @@ $(GOBIN)/goveralls: | $(BASE) ; $(info  building goveralls...)
 
 .PHONY: lint
 lint: | $(BASE) $(GOLINT) ; $(info  running golint...) @ ## Run lint test
+	$Q mkdir -p $(BASE)/test
```

Comment:
we should move to golangci-lint in different PR

Reply:
this is to avoid posting output to `/dev/stderr` as it does not work as a regular user on some setups.
the goal is to be able to do `make * ` not as root.

i did not change the linter